tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+companies, campaigns and ads. Once you have these files, you can go ahead and load them using the standard PostgreSQL :code:`\COPY` command.
+
+::
+                                                                                          
+    \copy companies from 'companies.csv' WITH CSV;                                                     
+    \copy campaigns from 'campaigns.csv' WITH CSV;                                                     
+    \copy ads from 'ads.csv' WITH CSV;
+
+Running queries and modifications
+---------------------------------
+
+Now that the data is loaded, let's go ahead and run some queries. You could start with a simple count
+query to see how many companies we have data for.              
+                                                                                          
+::
+
+    SELECT count(*) from companies;                                                           
+                                                                                          
+A more interesting query for a company to run would be to see details about its campaigns with maximum budget.
+We could do that by running the below query.                                     
+
+::
+                                                                                          
+    SELECT name, cost_model, state, monthly_budget
+    FROM campaigns
+    WHERE company_id = 5
+    ORDER BY monthly_budget DESC
+    LIMIT 10;
+                                                                                          
+Next, we can also run a join query across multiple tables to see information about running campaigns which receive the most clicks and impressions.
+
+::
+                                                                                          
+    SELECT campaigns.id, campaigns.name, campaigns.monthly_budget,
+           sum(impressions_count) as total_impressions, sum(clicks_count) as total_clicks
+    FROM ads, campaigns                                                                       
+    WHERE ads.company_id = campaigns.company_id                                               
+    AND campaigns.company_id = 5                                                              
+    AND campaigns.state = 'running'                                                           
+    GROUP BY campaigns.id, campaigns.name, campaigns.monthly_budget                           
+    ORDER BY total_impressions, total_clicks;                                                 
+                                                                                          
+Other than being able to run ad-hoc SQL for analytics, you can also run standard UPDATE and DELETE commands on your
+distributed table. For eg. if you want to update your budget for a particular campaign, you can do something like:
+
+::                                                                                          
+    
+    UPDATE campaigns
```

Comment:
This command only modifies a single row. What if we updated for all records for company_id = 5?